projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -161,11 +161,25 @@ public function initial_state() {
 		);
 	}
 
+	/**
+	 * Checks to see if the current post supports Publicize
+	 *
+	 * @return boolean True if Publicize is supported
+	 */
+	public function is_supported_post() {
+		$post_type = get_post_type();
+		return ! empty( $post_type ) && post_type_supports( $post_type, 'publicize' );
+	}
+
 	/**
 	 * Enqueue block editor scripts and styles.
 	 */
 	public function enqueue_block_editor_scripts() {
-		if ( ! ( new Modules() )->is_active( self::JETPACK_PUBLICIZE_MODULE_SLUG ) || class_exists( 'Jetpack' ) ) {
+		if (
+			! ( new Modules() )->is_active( self::JETPACK_PUBLICIZE_MODULE_SLUG ) ||
```

Comment:
Shouldn't the check for the publicize module being active alone would suffice here? Do we need the check for `is_supported_post` as well?

Reply:
We need the Jetpack check as otherwise we load the Jetpack version of the extension as well as ours. This could be an extra change that's crept into this PR though!

Also we don't want it being loaded for all post types, as this is what's breaking the site editor.

Reply:
The Jetpack check is from #24521